packages/picasso/src/Input/Input.tsx

```diff
@@ -156,10 +174,63 @@ const getMultilineLabel = ({
   return 'characters entered'
 }
 
+const MultilineStatusAdornment = (props: MultilineStatusAdornmentProps) => {
```

Comment:
It is more about multiline, right? What about
```suggestion
const MultilineAdornment = (props: MultilineAdornmentProps) => {
```